ZXing.Net.MAUI/ZXingNetExtensions.cs

```diff
@@ -4,26 +4,34 @@
 
 namespace ZXing.Net.Maui
 {
-	public static class ZXingNetExtensions
-	{
-		public static BarcodeResult ToBarcodeResult(this ZXing.Result result)
-			=> new()
-			{
-				Raw = result.RawBytes,
-				Value = result.Text,
-				Format = (BarcodeFormat)(int)result.BarcodeFormat,
-				Metadata = new Dictionary<MetadataType, object>(result?.ResultMetadata?.Select(md => new KeyValuePair<MetadataType, object> ((MetadataType)md.Key, md.Value))),
-				PointsOfInterest = result?.ResultPoints?.Select(p => new PointF(p.X, p.Y))?.ToArray()
-			};
+    public static class ZXingNetExtensions
+    {
+        public static BarcodeResult ToBarcodeResult(this ZXing.Result result)
+            => new()
+            {
+                Raw = result.RawBytes,
+                Value = result.Text,
+                Format = (BarcodeFormat)(int)result.BarcodeFormat,
+                Metadata = new Dictionary<MetadataType, object>(result?.ResultMetadata?.Select(md => new KeyValuePair<MetadataType, object>((MetadataType)md.Key, md.Value))),
+                PointsOfInterest = result?.ResultPoints?
+                    .Where(p => p is not null)
+                    .Select(p => new PointF(p.X, p.Y))?.ToArray()
+            };
 
-		public static BarcodeResult[] ToBarcodeResults(this ZXing.Result[] results)
-			=> results?.Select(result => new BarcodeResult()
-			{
-				Raw = result.RawBytes,
-				Value = result.Text,
-				Format = (BarcodeFormat)(int)result.BarcodeFormat,
-				Metadata = new Dictionary<MetadataType, object>(result?.ResultMetadata?.Select(md => new KeyValuePair<MetadataType, object>((MetadataType)md.Key, md.Value))),
-				PointsOfInterest = result?.ResultPoints?.Select(p => new PointF(p.X, p.Y))?.ToArray()
-			})?.ToArray();
-	}
+        public static BarcodeResult[] ToBarcodeResults(this ZXing.Result[] results)
+            => results?.Select(result => new BarcodeResult()
```

Comment:
Perhaps you could make this instead use `result => result.ToBarcodeResult()` reusing the other extension method instead of duplicating code?

Reply:
Hi Redth.
The format of my visual estudio on mac is using a different code style :(
The only change I made was to add `.Where(p => p is not null)`, 
Sorry for the bad formatting, it doesn't make the change clear, I can fix it...

Reply:
Hi @Redth, any update on this?